nrgpy/utils/utilities.py

```diff
@@ -111,17 +104,42 @@ def date_check(start_date, end_date, string):
         start = start_date
         end = end_date
 
-    try:
-        date_text = re.search(date_format, string)
-        file_date = datetime.strptime(date_text[0], strp_format)
+    if re.search(date_format_no_dash, string):
+        date_text = re.search(date_format_no_dash, string)
+        try:
+            file_date = datetime.strptime(date_text[0], strp_format_no_dash)
+            if (file_date >= start) and (file_date <= end):
+                return True
+            else:
+                return False
+        except ValueError:
+            ext_date_format_no_dash = "([0-9]{6}[0-9]{4}[0-9]{2}[0-9]{2})"
```

Comment:
do we want another try/except here, in case there ends up being a different 14-digit string in a filename? or just let it break if the filenames get so unruly?

Reply:
after that it's unsupported. i say let it break.